pom.xml

```diff
@@ -519,6 +536,21 @@
             <id>clearvolume</id>
             <url>http://dl.bintray.com/clearvolume/ClearVolume</url>
         </repository>
+        <repository>
```

Comment:
I'm not sure about Jitpack, especially when it comes to reproducible builds. Can you deploy your artifacts to Maven Central e.g.?

Reply:
I have been using it extensively for all my projects since quite a while.. only two downtimes in a couple of years

The problem with Maven is that the process is quite an hassle and I have absolutely not experience with that..